Packs/CommonScripts/Scripts/URLEncode/URLEncode.py

```diff
@@ -0,0 +1,26 @@
+import demistomock as demisto
+from CommonServerPython import *
+from urllib.parse import quote, unquote
+
+''' MAIN FUNCTION '''
+
+
+def main(args):
+    value = args.get('value')
+    decoded_value = unquote(value)
+    processed_value = quote(decoded_value)
+
+    eContext = {
+        'EncodedURL': processed_value
+    }
+
+    return CommandResults(readable_output=processed_value,
```

Comment:
a transformer shouldn't have context and readable output.
no need to use `CommandResults`